packages/eslint-plugin/src/rules/no-base-to-string.ts

```diff
@@ -46,6 +47,12 @@ export default createRule<Options, MessageIds>({
         type: 'object',
         additionalProperties: false,
         properties: {
+          checkUnknown: {
+            type: 'boolean',
+            default: false,
+            description:
+              'Checks the case where toString is applied to unknown type',
```

Comment:
[Docs] A little bit of phrasing alignment:

```suggestion
              'Whether to also check values of type `unknown`',
```